azure-pipelines.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 #      displayName: 'Run Python lint'
 
   - bash: |
-      make -C hw regs && git diff --exit-code
+      make -C hw regs && git restore -- hw/ip/gpio hw/ip/uart hw/ip/rv_timer && git diff --exit-code
```

Comment:
How about removing these modules from the `hw` makefile instead of reverting the check here?

And please explicitly comment them out and add a comment referring to an issue explaining what's going on. Autogenerated files in the tree which are actually not auto-generated are problematic.

Reply:
Thanks. I've changed `hw/Makefile` rather than azure-pipelines.yml